src/components/home-button/HomeButton.jsx

```diff
@@ -3,13 +3,27 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import {TYPE} from 'logo/Logo';
 
+const BASE_URL = 'https://styleguide.brainly.com/images/logos/';
+const LOGOS = {
+  'brainly': 'brainly-761d75d6ea',
+  'eodev': 'eodev-1972bd4349',
+  'nosdevoirs': 'nosdevoirs-e2d5d17215',
+  'znanija': 'znanija-addd85e6f5'
+};
+const ICONS = {
+  'brainly': 'brainly-mobile-426ef8718f',
+  'eodev': 'eodev-mobile-b3319881d5',
+  'nosdevoirs': 'nosdevoirs-mobile-0920e17308',
+  'znanija': 'znanija-mobile-4b7ac3cb04'
+};
```

Comment:
we had to hardcode this stuff in order for this component to work in other applications. Thankfully this data doesn't change much.